libsolidity/experimental/ast/TypeSystem.h

```diff
@@ -136,12 +136,14 @@ class TypeSystem
 	{
 		return m_typeClasses.at(_class.m_index).typeVariable;
 	}
-private:
-	friend class TypeEnvironment;
+
 	TypeClassInfo const& typeClassInfo(TypeClass _class) const
 	{
 		return m_typeClasses.at(_class.m_index);
 	}
```

Comment:
Not sure why this was private in the first place. I need to access it to get at the type variable after the class is already registered. Is there some other way I am supposed to get it?